docs/getting-started-guides/kubeadm.md

```diff
@@ -165,7 +167,8 @@ also support [Network Policy](/docs/user-guide/networkpolicies/). See the [add-o
 
 You can install a pod network add-on with the following command: 
 
-    # kubectl apply -f <add-on.yaml>
```

Comment:
Yes. This was changed in #1549 as we discussed, along with the reference to Weave and replaced with the lines below.